src/Logger/Logger.php

```diff
@@ -85,4 +96,15 @@ public static function hasProvider(string $providerName): bool
 
         return false;
     }
+
+    /**
+     * Return only a sample of the logs from this logger
+     *
+     * @param  float  $sample Total percentage of issues to use with 100% being 1
+     * @return void
+     */
+    public function sample(float $sample)
+    {
+        $this->samplePercent = $sample * 100;
```

Comment:
if the method is `sample` we can also use the same name for the attribute in sake of consistency.

Reply:
We could also add getSample for convenience. In this case, the method name we have can be changed to setSample.